tests/admin_utils/test_logentry.py

```diff
@@ -163,6 +163,26 @@ def test_logentry_unicode(self):
         log_entry.action_flag = 4
         self.assertEqual(six.text_type(log_entry), 'LogEntry Object')
 
+    def test_custom_log_actions(self):
```

Comment:
This file is mostly for unit testing the `LogEntry` model, so I would limit the tests to `LogEntryManager.log_action()`. Similarly, could you try to unit test the `ModelAdmin` methods rather than having complete request/response tests? Maybe in `tests/modeladmin`.

p.s. when sending or updating a pull request, please also update the ticket as I just did.